modules/33acrossBidAdapter.js

```diff
@@ -38,38 +41,37 @@ function _createServerRequest(bidRequest) {
     }
   }
 
-  // Allowing site to be a test configuration object or just the id (former required for testing,
-  // latter when used by publishers)
-  ttxRequest.site = params.site || { id: params.siteId };
+  ttxRequest.site = { id: params.siteId };
 
```

Comment:
Removed the ability to override site object in `ttxRequest`. That option was provided solely to set test endpoint which is no longer possible. The tester must set `test: 1` in order use test endpoint

Reply:
👍